### DISCUSSION TITLE
Scheduled zones and masks

### DISCUSSION THREAD

**Discussion by fyrstyk** [blakeblackshear/frigate#6010] [Ideas]
So, I now have a new PTZ camera.

During the day it sits in one position, and another position at night, so the zones/ masks are different sizes

I'd love the ability to be able to schedule a zone based on time.

Would that be possible?

**Comment by NickM-27** [comment]
probably some way to manually automate it now between 2 different configs, but this is a requested feature already https://github.com/blakeblackshear/frigate/issues/3621
